command/pipelines/pipeline_list_test.go

```diff
@@ -0,0 +1,155 @@
+package pipelines
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/spinnaker/spin/command"
+)
+
+func TestPipelineList_basic(t *testing.T) {
+	ts := testGatePipelineListSuccess()
+	defer ts.Close()
+
+	meta := command.ApiMeta{}
+	args := []string{"--application", "app", "--gate-endpoint", ts.URL}
+	cmd := PipelineListCommand{
+		ApiMeta: meta,
+	}
+	ret := cmd.Run(args)
+	if ret != 0 {
+		t.Fatalf("Command failed with: %d", ret)
+	}
+}
+
+func TestPipelineList_flags(t *testing.T) {
+	ts := testGatePipelineListSuccess()
+	defer ts.Close()
+
+	meta := command.ApiMeta{}
+	args := []string{"--gate-endpoint", ts.URL} // Missing application.
+	cmd := PipelineListCommand{
+		ApiMeta: meta,
+	}
+	ret := cmd.Run(args)
+	if ret == 0 { // Success is actually failure here.
```

Comment:
Can you explain why?

Reply:
Just asking because in the other tests where you do it "malformed" or "fail" is in the title and that makes it a little clearer.